Libraries/Components/Button.js

```diff
@@ -25,107 +25,210 @@ import type {ColorValue} from '../StyleSheet/StyleSheet';
+    the [Android
+    documentation](https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusRight).
+
+    @platform android, tv
    */
   nextFocusRight?: ?number,
 
   /**
-   * TV next focus up (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates up. See
+    the [Android
+    documentation](https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusUp).
+
+    @platform android, tv
    */
   nextFocusUp?: ?number,
 
   /**
-   * Text to display for blindness accessibility features
+    Text to display for blindness accessibility features.
    */
   accessibilityLabel?: ?string,
 
   /**
-   * If true, disable all interactions for this component.
+    If `true`, disable all interactions for this component.
+
+    @default false
    */
   disabled?: ?boolean,
 
   /**
-   * Used to locate this view in end-to-end tests.
+    Used to locate this view in end-to-end tests.
    */
   testID?: ?string,
 |}>;
 
 /**
- * A basic button component that should render nicely on any platform. Supports
- * a minimal level of customization.
- *
- * <center><img src="img/buttonExample.png"></img></center>
- *
- * If this button doesn't look right for your app, you can build your own
- * button using [TouchableOpacity](docs/touchableopacity.html)
- * or [TouchableNativeFeedback](docs/touchablenativefeedback.html).
- * For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js).
- * Or, take a look at the [wide variety of button components built by the community](https://js.coach/react-native?search=button).
- *
- * Example usage:
- *
- * ```
- * import { Button } from 'react-native';
- * ...
- *
- * <Button
- *   onPress={onPressLearnMore}
- *   title="Learn More"
- *   color="#841584"
- *   accessibilityLabel="Learn more about this purple button"
- * />
- * ```
- *
+  A basic button component that should render nicely on any platform. Supports a
+  minimal level of customization.
+
+  If this button doesn't look right for your app, you can build your own button
+  using [TouchableOpacity](touchableopacity) or
+  [TouchableWithoutFeedback](touchablewithoutfeedback). For inspiration, look at
+  the [source code for this button
+  component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js).
+  Or, take a look at the [wide variety of button components built by the
+  community](https://js.coach/?menu%5Bcollections%5D=React%20Native&page=1&query=button).
+
+  ```jsx
+  <Button
+    onPress={onPressLearnMore}
+    title="Learn More"
+    color="#841584"
+    accessibilityLabel="Learn more about this purple button"
+  />
+  ```
+
+  @example ```SnackPlayer name=Button%20Example
+  import React from 'react';
+  import { StyleSheet, Button, View, SafeAreaView, Text, Alert } from 'react-native';
```

Comment:
@motiz88 I haven't wrapped any line of code present in the comments as I think this would cause unnecessary complications when we parse them. Also, they would require manual wrapping if we decide doing this. What do you suggest?

>This line is 84 characters.

Reply:
That makes sense 👍🏻